lms/urls.py

```diff
@@ -919,6 +919,10 @@
     )
 
 urlpatterns += (
+    url(r'^labster_license/', include('labster_course_license.webhook_urls')),
```

Comment:
because we will have url like `api/courses/some_course_id/license_update` then. It's ok for edx but not for API webhook because of `some_course_id`